server-model.js

```diff
@@ -0,0 +1,25 @@
+const db = require('./data/dbConfig.js');
+
+module.exports = {
+    get,
+    findById,
+    add,
+}
+
+function get() {
+    return db('games')
+};
+
+function findById(id) {
+    return db('games')
+    .where({ id })
+    .first()
+};
+
+function add(games) {
+    return db('games')
+    .insert(games)
+    .then(id => {
+        return findById(id[0])
```

Comment:
you could use the .first() here in place of the [0] to remain consistent with the previous helper method